build.sbt

```diff
@@ -39,10 +38,41 @@ val backend = project
     )
   )
 
+// Needs to be called `jsonConfig` because of naming conflict with sbt universe...
```

Comment:
You mean it'd conflict with `bloop-config`?

Reply:
No, i mean that the name `config` is imported implicitly by sbt, so we cannot call it `config`, we need to prepend json.